docs/requirements.txt

```diff
@@ -2,3 +2,4 @@ sphinx
 sphinx_rtd_theme
 recommonmark
 breathe
+reno[sphinx]==3.5.0
```

Comment:
Just noting that in cloud client we use version 3.4.0, not that I have anything against 3.5.0

Reply:
And 3.3 in dwave-samplers and dimod :smile: